docs/A-Introduction/information.md

```diff
@@ -78,33 +87,44 @@ To learn how to convert between hexadecimal and binary refer to the chapter enti
 
 The memory model for organizing information stored in RAM is linear.  Any byte in memory is accessible through a map that treats each actual physical memory location as a position in a continuous sequence of locations aligned next to one another.
 
+<span className="mdImg" align="center" alt="A memory model image">
+
 ![RAM memory model: linearly mapped \(continuous sequence\)](https://ict.senecacollege.ca//~ipc144/pages/images/ram.png)
```

Comment:
This image needs to be brought into the repo

Reply:
I have modified it, thx.

Reply:
I don't understand, it's still pointing to https://ict.senecacollege.ca//~ipc144/pages/images/ram.png.  All images need to get imported into this repo and added to git.